### ISSUE TITLE
V6.1.1 init with pagination, pagination DOM is empty

### ISSUE THREAD

**Issue by believeZJP** [nolimits4web/swiper#3768]
This is a (multiple allowed):
* [x] bug

* Swiper Version: EXACT RELEASE VERSION OR COMMIT HASH, HERE.

**V6.1.1**

* Platform/Target and Browser Versions: PLATFORM CLIENT YOU ARE TARGETING SUCH AS macOS, Windows, CORDOVA, IOS, ANDROID, CHROME, ETC.

**macOS, CHROME**

* Live Link or JSFiddle/Codepen or website with isssue: PREFERABLY _(IF YOU WANT YOUR ISSUE TO BE RESOLVED ASAP)_.

[https://codepen.io/believezjp/pen/xxVOboe](https://codepen.io/believezjp/pen/xxVOboe)

### What you did
EXPLAIN WHAT YOU DID, PREFERABLY WITH CODE EXAMPLES, HERE.
```js
new Swiper('.swiper-container', {
        // need pagination
        pagination: {
          el: '.swiper-pagination',
          type: 'bullets'
       }
})
```
**use Swiper in a vue.js project, init with `pagination`, check  DOM `<div class="swiper-pagination"></div>` there is no bullets in pagination DOM.**

### Expected Behavior
EXPLAIN WHAT IS TO BE EXPECTED, HERE.

**show pagination bullets DOM in `<div class="swiper-pagination"></div>`**

### Actual Behavior
EXPLAIN WHAT IS ACTUALLY HAPPENING, HERE.

P.S. Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/swiper)
for that.

Before you open an issue, please check if a similar issue already exists or has been closed before.

**Comment by simonmaass** [comment]
i am having the same problem...!!

**Comment by simonmaass** [comment]
I think this issue is related to vue-awesome-swiper....

**Comment by believeZJP** [comment]
> I think this issue is related to vue-awesome-swiper....

No, You can try with `import Swiper from 'swiper'` in your project, it's the same problem. It's not not  caused by `vue-awesome-swiper`.

**Comment by aKzenT** [comment]
I'm having the same problem, but after updating from 5.3.6 to 5.4.5. Pagination bullets are no longer shown

**Comment by GitOfZGT** [comment]
> 我遇到了同样的问题，但是从5.3.6更新到5.4.5之后。分页符号不再显示

I known why!

**Comment by GitOfZGT** [comment]
I known why!

![image](https://user-images.githubusercontent.com/21262000/91318310-aa617300-e7ed-11ea-85d2-1f67e0237a51.png)
![image](https://user-images.githubusercontent.com/21262000/91318446-cfee7c80-e7ed-11ea-815a-7b75e7a10add.png)

but  `import Swiper from "swiper/swiper-bundle.esm.js"`  is  ok!

**Comment by simonmaass** [comment]
@nolimits4web can this be reopened?

**Comment by dan2k3k4** [comment]
Also getting the same issue but did the change suggested by @GitOfZGT and worked fine:

```diff
-import Swiper from 'swiper'
+import Swiper from 'swiper/swiper-bundle.esm'
```

**Comment by sergey2mc** [comment]
I have the same issue but on the latest 8.3.2 in Angular...
I have 1 slider per page.
Initially, pagination gets shown correctly.
However, when I get navigated to some other page, the pagination is missing.

I solved the issue by specifying `uniqueNavElements: false` in the Swiper options.

**Comment by TonyCh55** [comment]
> I have the same issue but on the latest 8.3.2 in Angular... I have 1 slider per page. Initially, pagination gets shown correctly. However, when I get navigated to some other page, the pagination is missing.
> 
> I solved the issue by specifying `uniqueNavElements: false` in the Swiper options.

You saved me thanks) In my case custom pagination worked in some pages and didn't work on other, fucking Swiper)

**Comment by TonyCh55** [comment]
> I have the same issue but on the latest 8.3.2 in Angular... I have 1 slider per page. Initially, pagination gets shown correctly. However, when I get navigated to some other page, the pagination is missing.
> 
> I solved the issue by specifying `uniqueNavElements: false` in the Swiper options.

@sergey2mc in my case pagination disappeared from DOM when i have 2 carousels on one page, and one carousel triggers the other on same page
